low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -64,7 +64,11 @@ def get_health_info(self, rpath):
         leaf_node, _ = self.get_node_details(nodes[-1])
         if leaf_node == "compute":
             for fru in self.server_frus:
-                info.update({fru: self.server_frus[fru]()})
+                try:
+                    info.update({fru: self.server_frus[fru]()})
+                except:
+                    # TODO: Log the exception
+                    info.update({fru: None})
```

Comment:
Added exception handling here to avoid breaking any other ongoing FRUs call.
cc: @sumedhak27